int32 action
---
float32[] state
float32 reward
bool done
string info